scalafmt-cli/src/main/scala/org/scalafmt/cli/CliArgParser.scala

```diff
@@ -99,8 +89,8 @@ object CliArgParser {
         .text(
           "use project filters even when specific files to format are provided"
         )
-      opt[String]('c', "config")
-        .action(readConfigFromFile)
+      opt[File]('c', "config")
```

Comment:
Do you know how this change affects parsing of relative paths? Does it absolutize eagerly? Ideally, relative filenames should be kept relative so that we can resolve them from the working directory that nailgun provides.

Reply:
`File` keeps them relative. `CommonOptions.cwd` will contain the nailgun directory, and lazy `CommonOptions.workingDirectory` is initialized from `cwd` or else from `user.dir`.